odh-jupyter-trash-cleanup/odh_jupyter_trash_cleanup/trash.py

```
@@ -0,0 +1,57 @@
+EMPTY_TRASH_COMAND = [GIO, "trash", "--empty"]
+LIST_TRASH_FILES_COMMAND = [GIO, "trash", "--list"]
+
+# For manual remotion of files
+XDG_DATA_HOME = os.environ.get("XDG_DATA_HOME")
+TRASH_DIR = (Path(XDG_DATA_HOME) if XDG_DATA_HOME else Path.home() / ".local" / "share") / "Trash"
+
+async def empty_trash(self) -> int:
+    """Remove all items from Trash, permanently deleting them"""
+    deleted = 0
+    if shutil.which(GIO) is not None:
+        count_files = subprocess.run(LIST_TRASH_FILES_COMMAND, capture_output=True, check=False)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)